src/components/ListMessages.tsx

```diff
@@ -0,0 +1,50 @@
+import { MessageResponse } from "@frequency-chain/api-augment/interfaces";
+import { Button, Space, Table } from "antd";
+import Column from "antd/lib/table/Column";
+import React from "react";
+import { fetchMessagesForSchema } from "../services/chain/apis/extrinsic";
+import { MessageDetails, SchemaProps } from "../services/types";
+import { staticSchema } from "./RegisterSchema";
+
+const ListMessages = (props: SchemaProps): JSX.Element => {
```

Comment:
Here as well. I don't think we would want to share props.

TS allows types to auto cast when they have the same signature. So a variable of type `interface X { x: bool }` can be used on a function with with a argument of type `interface NotX {x: bool }`